### ISSUE TITLE
Wireless Storage Controller out of chunk usage

### ISSUE THREAD

**Issue by Aster0** [Aster0/UnifiedStorage#4]
Currently, wireless storage controllers are not usable when far away out of chunk from the linked Storage Controller.

**Comment by drosofsky** [comment]
will this issue be being resolved?

**Comment by Aster0** [comment]
> will this issue be being resolved?

Yes, it's in progress. However, it's not currently the priority. 

You can see my priority list here: https://github.com/users/Aster0/projects/2
